pkgs/top-level/all-packages.nix

```diff
@@ -15347,7 +15347,8 @@ in
 
     phc-intel = callPackage ../os-specific/linux/phc-intel { };
 
-    prl-tools = callPackage ../os-specific/linux/prl-tools { };
+    # Disable for kernels 4.15 and above due to compatibility issues
+    prl-tools = if stdenv.lib.versionOlder kernel.version "4.15" then null else callPackage ../os-specific/linux/prl-tools { };
```

Comment:
Only noticed because my configuration didn't eval ;), this condition I think is wrong (or the null should be on the other side of the branch or something).

Reply:
correct, I swapped the callPackage and null